examples/jest/jest.bzl

```diff
@@ -10,8 +10,6 @@ def jest_test(name, srcs, deps, jest_config, **kwargs):
         "--ci",
     ]
     args.extend(["--config", "$(location %s)" % jest_config])
```

Comment:
I was expecting to see a glob pattern being added here or in the place jest_test is called

Reply:
The glob pattern was already present within `jest.config.js` (that's the idiomatic way of specifying tests to jest). Do you have a different API you would prefer?